### PR TITLE
COTECH-1025 Track eligible event for connatix

### PR DESCRIPTION
**Pull request by andrzejkupczyk**
https://fandom.atlassian.net/browse/COTECH-1025

### CODE REVIEW

File: src/ad-services/connatix/connatix-tracker.ts [hunk 1]
```diff
@@ -44,6 +44,11 @@ export class ConnatixTracker implements VideoTracker {
 		this.track(eventName, VideoEventProvider.getEventData(this.getVideoData(eventName)));
 	}
 
+	trackEligible() {
+		const eventName = 'eligible';
```
Comment (latata) [inline src/ad-services/connatix/connatix-tracker.ts:48]: redundant var ;)

**Review by latata — APPROVED** [review]

File: src/ad-services/connatix/connatix-tracker.ts (hunk 1 above)
Comment (andrzejkupczyk) [reply src/ad-services/connatix/connatix-tracker.ts:48]: Objectively speaking, that is not true :)

**Review by dzawada — DISMISSED** [review]

File: src/ad-services/connatix/connatix-tracker.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ const logGroup = 'connatix';
 export class ConnatixTracker implements VideoTracker {
 	private playerApi: ConnatixPlayerApi;
 
-	private trackingEvents = {
+	private playerEvents = {
```
Comment (nandy-andy) [inline src/ad-services/connatix/connatix-tracker.ts:10]: If we want to rename the property then I think both `trackingEvents` and `playerEvents` are equally wrong :D It's rather `playerEventsToTrackingEventsMap` 😆 The keys are names of the events emitted by the player whereas the values are the events/values we store in our DWH.

Comment (andrzejkupczyk) [reply src/ad-services/connatix/connatix-tracker.ts:10]: The context is connatix TRACKER, so adding "to tracking" seems redundant. I can update the name if needed. What's you final verdict? :)

Comment (nandy-andy) [reply src/ad-services/connatix/connatix-tracker.ts:10]: We've discussed it on slack. It was faster this way :)

**Review by nandy-andy — APPROVED** [review]

**Pull request merged**